package.json

```diff
@@ -30,6 +31,7 @@
     "typescript": "~3.7.2"
   },
   "devDependencies": {
+    "@types/lodash": "^4.14.161",
```

Comment:
Shouldn't lodash in itself also be a dependency?

Reply:
Lodash is included in CRA.

Reply:
If we use it, it is an explicit dependency and should be listed.

Reply:
Cool, I'll add it then. I'm a huge lodash fan because it usually does stuff faster than native. 

A little bit of trivia about it, the author of lodash worked on the Chakra team for a while (Explorer's JS interpreter) and it was during the time when Chakra was faster than V8 for most things. I think V8 is back on top now, though and idk where he went